chat-plugins/modlog.js

```diff
@@ -6,7 +6,10 @@ const ProcessManager = require('./../process-manager');
 const execFileSync = require('child_process').execFileSync;
 
 const MAX_PROCESSES = 1;
-const RESULTS_MAX_LENGTH = 100;
+const DEFAULT_RESULTS_LENGTH = 100;
+const MORE_BUTTON_INCREMENTS = [200, 400, 800, 1600, 3200];
+const LINES_SEPARATOR =	'lines=';
```

Comment:
I'm surprised we don't have an eslint rule against this. I'm against this kind of alignment.